_posts/2019-12-18-sentry-the-guardian-of-your-quarkus-app.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: 'Sentry error tracker - The guardian of your Quarkus application'
+date: 2019-12-18
+tags: quarkus extension logging cloud
+author: adamevin
+---
+
+:imagesdir: /assets/images/posts/quarkus-sentry
+
+*If like me you have met Quarkus and fell in love*, you started implementing your backend and you started to feel so light and quick, click clack, whoot whoot... *in a woosh, your lightning fast native container is ready to be sent on your brand new kubernetes cluster 🤙*
+
+*This is awesome!!*
+
+Mmmeahh *you may have forgotten a detail...* have you thought about *the fate of your first production exception* (please be honest with me :-)?
+
+["upperalpha"]
+. As a backend god, master of the world, first of his name, you implemented some logstash process to extract your logs files, send them to some elastic search server somewhere and... you just spend more time configuring it than actually coding your app 😂.
+. You thought you could manually check the error logs a few times a week and ... oups, so unlucky, your application broke exactly the day you forgot to check, let's hope no one else noticed 😅 ?
+. *You were waiting for this article 🤓*
+
+Now that you are teased, aren't you? Let's dive in. 
+
+== Sentry, the guardian has arrived on Quarkus
+
+Once your code is reviewed, tested in many ways, containerized, ..., and finally *promoted to production*, [line-through]#I guess you can expect it to run smoothly ever after..# *an exception is going to pop*, it's just a question of time. The questions are:
+
```

Comment:
will have to mention their recent license changes so we don't end up recommending this as something to be usable in opensource blindly.

Reply:
Yes, thanks for the info @maxandersen, Good catch !!